packages/cli/src/cmds/beacon/initBeaconState.ts

```diff
@@ -44,6 +44,14 @@ export async function initBeaconState(
     return await initFromFile(args.weakSubjectivityStateFile);
   } else if (dbHasSomeState) {
     return await initStateFromDb(config, db, logger);
+  } else if (args.fetchWeakSubjectivityStateFromIPFS) {
+    const stateData = readFile("./weakSubjectivityState.json") as {ipfsPath: string; stateRoot: string};
```

Comment:
Why would this file `./weakSubjectivityState.json` exist?

Reply:
as it stands right now, it would be because we copied it from the output of beacon-state-upload (from here: https://github.com/ChainSafe/beacon-state-upload/pull/3/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R39)
perhaps we can have a more automated process, but that's the way it is right now

Reply:
I still don't follow why the file should exist in any environment that you may run the cli.

Reply:
Two reasons
1) so we can have the state root of the weak subjectivity state to check against when downloading from IPFS so as to not rely purely on the security of the SSL certs (as discussed on one of the eth2 implementer's calls about Teku's state loading process from Infura)
2) so we can have the IPFS link from which to download the weak subjectivity state

Maybe it's better to just embed directly into the code though rather than rely on a file.  We'll just have to remember to update the code for every release that we have a new weak subjectivity state, but that's just as much of a process as remembering to copy the file anyway.

Reply:
But again if we add the file at the project root `/weakSubjectivityState.json`, and you run the command from the root it will work. If you run the command from `/packages/cli` it will break since `./weakSubjectivityState.json` does not exist

Reply:
ahh ok i gotcha.  yeah i'll move it into `/src`